README.rst

```diff
@@ -0,0 +1,320 @@
+==============================================
+ REANA example - LHCb Rare Charm Decay Search
+==============================================
+
+.. image:: https://img.shields.io/travis/reanahub/reana-demo-lhcb-d2pimumu.svg
+   :target: https://travis-ci.org/reanahub/reana-demo-lhcb-d2pimumu
+
+.. image:: https://badges.gitter.im/Join%20Chat.svg
+   :target: https://gitter.im/reanahub/reana?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge
+
+.. image:: https://img.shields.io/github/license/reanahub/reana-demo-lhcb-d2pimumu.svg
+   :target: https://raw.githubusercontent.com/reanahub/reana-demo-lhcb-d2pimumu/master/LICENSE
+
+
+
+About
+======
+
+This analysis example attempts to reproduce `LHCb rare charm decay study <https://cds.cern.ch/record/1543929>`_
+published in `Phys. Lett. B 724 (2013) 203-212 <https://www.sciencedirect.com/science/article/pii/S0370269313004747?via%3Dihub>`_.
+
+This study was done on decays of:
+
+<a href="https://www.codecogs.com/eqnedit.php?latex=$D_{(s)}^{&plus;}\rightarrow&space;\pi^{&plus;}&space;\mu^{&plus;}&space;\mu&space;^{-}$" target="_blank"><img src="https://latex.codecogs.com/gif.latex?$D_{(s)}^{&plus;}\rightarrow&space;\pi^{&plus;}&space;\mu^{&plus;}&space;\mu&space;^{-}$" title="$D_{(s)}^{+}\rightarrow \pi^{+} \mu^{+} \mu ^{-}$" /></a>
+
+These decays are very rare, which makes their observation extremely
+challenging. The LHC produces a lot of charm particles (_D_), but it also
+produces a much greater number of other particles which can be mistaken for the
+signal. It is necessary to develop an effective strategy to identify the signal
+events in the large data sample. The event selection strategy is implemented in
+three stages: the trigger selection, stripping selection and the multivariate
+analysis.
+
+In the [paper](https://cds.cern.ch/record/1543929), you can lean more about the
+theoretical background and motivation to study this decay.
+
+This example is based on this
+`analysis-case-study <https://github.com/atrisovic/analysis-case-study>`_.
+
+Analysis structure
+===================
+
+Making a research data analysis reproducible basically means to provide
+"runnable recipes" addressing (1) where is the input data, (2) what software was
+used to analyse the data, (3) which computing environments were used to run the
+software and (4) which computational workflow steps were taken to run the
+analysis. This will permit to instantiate the analysis on the computational
+cloud and run the analysis to obtain (5) output results.
+
+
+1. Input data
+-------------
+The data being used is currently only available on request. It consists of a 10
```

Comment:
You can delete separate `README.md` in the `data` directory, since we mention this here already.